domain/life.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package domain
+
+// Life represents the life of an entity
+// as recorded in the life lookup table.
```

Comment:
That's not the point here. We have `core/life` already. This is so that when we load do rows which have the life int FK, we can have consts to deal with that.

Reply:
This is moot now that the watcher has been restructured.